src/osx/carbon/Info.plist.in

```diff
@@ -1,47 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist SYSTEM "file://localhost/System/Library/DTDs/PropertyList.dtd">
-<plist version="0.9">
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
 <dict>
-	<key>NSPrincipalClass</key>
-	<string>wxNSApplication</string>
-	<key>CFBundleInfoDictionaryVersion</key>
-	<string>6.0</string>
-	<key>CFBundleIdentifier</key>
-	<string>org.wxwindows.IDENTIFIER</string>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
-	<key>CFBundleLocalizations</key>
-	<array>
-		<string>de</string>
-		<string>en</string>
-		<string>fr</string>
-		<string>it</string>
-	</array>
 	<key>CFBundleExecutable</key>
-	<string>EXECUTABLE</string>
+	<string>${MACOSX_BUNDLE_EXECUTABLE_NAME}</string>
+	<key>CFBundleGetInfoString</key>
+	<string>${MACOSX_BUNDLE_INFO_STRING}</string>
 	<key>CFBundleIconFile</key>
 	<string>wxmac.icns</string>
```

Comment:
perhaps we should use MACOSX_BUNDLE_ICON_FILE here as well in the template ?

Reply:
Setting it to anything other than `wxmac.icns` is not going to work because the makefile rules always copy the actual icon being used to `Resources/wxmac.icns`, so I decided not to pretend that it can be done, when it can't.